lib/_http_server.js

```diff
@@ -475,6 +473,7 @@ function onParserExecuteCommon(server, socket, parser, state, ret, d) {
     socket.removeListener('data', state.onData);
     socket.removeListener('end', state.onEnd);
     socket.removeListener('close', state.onClose);
+    socket.removeAllListeners('drain');
```

Comment:
.removeAllListeners() also removes any user-installed 'drain' event listeners so no, probably not a good idea.